homeassistant/components/doorbird.py

```diff
@@ -24,14 +25,20 @@
 CONF_DOORBELL_EVENTS = 'doorbell_events'
 CONF_CUSTOM_URL = 'hass_url_override'
 
+DEVICES_SCHEMA = vol.All(cv.ensure_list,
```

Comment:
Call it `DEVICE_SCHEMA` and move the `vol.All(cv.ensure_list, [DEVICE_SCHEMA])` part to the `CONFIG_SCHEMA`.